frontend/src/layouts/SidebarLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import { Outlet } from "react-router-dom";
+import { Divider, Grid } from "@mui/material";
+
+import { NavigationBar } from "../components/NavigationBar";
+
+export const SidebarLayout = () => {
+  return (
+    <Grid container>
+      <Grid item xs={2}>
```

Comment:
The sidebar width should probably have a maximum value, otherwise it looks rather weird on wide screens. (We can fix this when we replace the navigation bar logo.)